JS/js1.js

```diff
@@ -0,0 +1,18 @@
+// // FORM SUBMIT
+
+let forms = document.querySelectorAll('form')
+
+for (let i = 0; i < forms.length; i++) {
+  forms[i].addEventListener('submit', event => {
+    event.preventDefault()
+
+    let input1 = forms[i]['textarea1']
```

Comment:
If you only have one form, you wouldn't need to iterate and then you could just access the information in the textareas like `form['textarea1']`